src/net/sourceforge/plantuml/gitlog/GitDiagram.java

```diff
@@ -93,7 +90,8 @@ protected ImageData exportDiagramInternal(OutputStream os, int index, FileFormat
 			margin1 = 10;
 			margin2 = 10;
 		}
-		final ImageParameter imageParameter = new ImageParameter(new ColorMapperIdentity(), false, null, dpiFactor, "",
+		// TODO change to other ImageParameter constructor?
+		final ImageParameter imageParameter = new ImageParameter(new ColorMapperIdentity(), false, null, "",
```

Comment:
GIT diagrams currently do not support Scale / DPI / Handwritten / Background color etc but its easy to enable them by changing to the other ImageParameter constructor here.   Just wanted to check first if thats ok?  Not sure if there was a special reason to use the current constructor here.

Reply:
Yes, do whatever you want with this :-)
This is a secret feature, no one is using is yet.
[For example, you can have](http://www.plantuml.com/plantuml/umla/HOxVJl9044VlUOfVyjq0-PIMyaUyCA1YY97eAupTsNPYcJMp2qh2mrkrnCl9d7DcbZ6HfKfIRmZA_6oQNsNe1vDAT5IwE98OzppFiW6UwCZ6xXft3PbuOOSoEBx4ltRTwkE25dvIJF_01_d4XmMjK0iRMLaVKPEwHhJgj1E6M6J5t4sB2VgRzUeU5pVOKKni_z7zyJiESBkNfis-M7XdQuw8KYcsT605MH9FPOhVtV6uTne1lJQT-p8lvwzdTHlscl38-YOQKOSTT_mG4yypvtC-eoldi7dUhi5wm84CAM3tSwiCcaYKhNENhAwIz0K0).
scale/DPI/Handwritten/background won't work because there is no instruction to change it for those diagrams.
But nevertheless, you can do this refactoring.